pages/mapping.js

```diff
@@ -101,32 +102,43 @@ export default function mapping() {
 
 	function handleNextSelected() {
 		var countryCounts = gameStats.country_counts;
+		const mappedCount = gameStats.mapped_count;
 		if (countryCounts[question.country_code]) {
 			countryCounts[question.country_code] = countryCounts[question.country_code] + 1;
 		} else {
 			countryCounts[question.country_code] = 1;
 		}
-		setGameStats({ country_counts: countryCounts });
+		setGameStats({ country_counts: countryCounts, mapped_count: mappedCount + 1 });
 
 		if (counter < questions.length) {
 			setCounter(counter + 1);
 			setLocationResults({ yes_count: 0, no_count: 0, maybe_count: 0, total_count: 0});
+	  	} else {
+	  		fetchUserStats();
+			fetchUntaggedLocationsCount();
+			fetchFact();
 	  	}
 	}
 
 	async function fetchUntaggedLocationsCount() {
 		// Get number of untagged locations
-		const result = await fetch(`/api/getUntaggedLocationsCount/${cookies.uuid}`);
+		const result = await fetch(`/api/getUntaggedLocationsCount/${cookies.uuid}`)
 		const response = await result.json();
 		setUntaggedLocationsCount(await response.count);
 	};
```

Comment:
We had an offline conversation about this, but we got the line numbers mixed up. This is the line where the `;` should be removed